tests/testthat/test-getConceptSetDefinitionBySourceKey.R

```diff
@@ -0,0 +1,7 @@
+with_mock_dir("mocks/getConceptSetDefinitionBySourceKey", {
+  
+  test_that("getConceptSetDefinitionBySourceKey works", {
```

Comment:
is it possible to have a test with `vocabularySourceKey` set to some value?

Reply:
Yep!